### DISCUSSION TITLE
How to use window.scrollTo?

### DISCUSSION THREAD

**Discussion by gittrolo** [quasarframework/quasar#14983] [General - Components / Directives / etc]
I have this method that I recall after closing a dialog:

start() {
	this.inputField= "";
	this.$nextTick(() => {
	this.$refs.ref_inputField.resetValidation();
	});

	//NOT working
	this.$nextTick(() => {
		window.scrollTo(0, 0);
	});

	//Working
	setTimeout(function () {
		window.scrollTo(0, 0);
	}, 1000);
},

window.scrollTo working only inside setTimeout. Why?

**Comment by suleimanas** [comment]
If I remember nextTick runs after updating the virtual DOM, before browser renders the changes

**Comment by pdanpdan** [comment]
The dialog locks the scroll while it is open, and a little bit after, that is why you need setTimeout

**Reply by gittrolo** [reply]
so the only solution  (very bad solution) is the setTimeout?

PS: Another strange behavior: if I press a button at the bottom of the page that first scrolls up and then opens a dialog. When I close the dialog it automatically goes back to the bottom where the pressed button is
